config/argo/argo.yaml

```diff
@@ -4,6 +4,12 @@ metadata:
   name: argo
   namespace: system
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: argo-server
```

Comment:
One change, we don't need an `argo-server`, let's remove all refs. for Argo server.

Reply:
Removed